lab_1/main.py

```diff
@@ -11,8 +11,16 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
 
+    if not isinstance(text, str):
+        return None
+    isalpha_text = ''
+    for i in text:
+        if i.isalpha() is True or i == ' ':
+            isalpha_text += i
+    isalpha_text = isalpha_text.lower()
+    tokens = isalpha_text.split()
+    return tokens
```

Comment:
you can join lower, split and return them just here

Reply:
done